api/localdev.env

```diff
@@ -1,3 +1,6 @@
+# Switch AUTH Providers ['KEYCLOAK','AWS_COGNITO']
```

Comment:
The front-end code has commented out all the keycloak logic, so being able to switch in the API won't accomplish much. However, this is fine.

Reply:
Yeah... I know, didn't communicate this to Catherine in the beginning that backend would provide a switch so she commented out every Keycloak out from frontend. It's my fault. 
It's ok, will create a ticket to address remaining Keycloak code cleaning after integration is doing good after deployment and having used for business (tech debt task)...

Reply:
Ticket for clean up is added (#311).